packages/now-client/src/utils/index.ts

```diff
@@ -125,6 +125,8 @@ export const fetch = async (
   // @ts-ignore
```

Comment:
I wonder if it would be possible to avoid all the `@ts-ignore` by using object spread:

```
opts.headers = {
  ...opts.headers,
  authorization: `Bearer ${token}`,
  accept: 'application/json',
  'user-agent': `now-client-v${pkg.version}`
};
```